api/stress_testing/README.md

```diff
@@ -0,0 +1,10 @@
+# Stress testing script
+
+This script runs `/works` and `/works/{id}` queries with a selection of query strings and work IDs, at a rate that is similar to a high-ish level of usage of the API.
```

Comment:
How were the work IDs selected? Is it random, or are they meant to be some “interesting” cross-sample of the data (e.g. a Miro record, a Sierra record, lots of items, no items)?

Reply:
Similar to the diff tool - just a human-random selection!

Reply:
Can you write that in this README please? :)